website/docs/reference/sinks/kafka.md

```diff
@@ -120,6 +120,21 @@ import CodeHeader from '@site/src/components/CodeHeader';
     
     # OPTIONAL - Key path
     key_path = "/path/to/host_certificate.key" # example, no default
+
+  # OPTIONAL - Sasl
+  [sinks.my_sink_id.sasl]
+    
+    # OPTIONAL - Enabled
+    enabled = true # default
+
+    # OPTIONAL - Username
+    username = "username" # example, no default
+
+    # OPTIONAL - Password
+    password = "password" # example, no default
+    
+    # OPTIONAL - mechanism
+    mechanism = "scram-sha-256" # example, no default
```

Comment:
This file is actually generated automatically from the data in `.meta/sinks/kafka.toml`, so if you add the new options there they will automatically get propagated where they need to go.

You can see an example of a config sub-group [here](https://github.com/timberio/vector/blob/e76083548a2d46664acd67a8e40f1835614d94c5/.meta/sinks/elasticsearch.toml#L47-L62).

Reply:
Hi,
Actually librdkafka return a error if mechanism not corrrect when healthcheck = true

About the mistake in documentation : I'll correct this commit with a 'amend' command this week-end

Reply:
If mechanism are invalid (only UPERCASE):

![image](https://user-images.githubusercontent.com/12722481/71218427-bc376180-22c1-11ea-8a7c-3a5cb9bf3936.png)